### DISCUSSION TITLE
Best way to limit elastalert metadata indices in Kibana?

### DISCUSSION THREAD

**Discussion by roman-tasi** [jertel/elastalert2#1355] [Q&A] [answered]
My elastalert-related indices seem to be growing indefinitely without any way to limit their growth. These are the indices:
```
elastalert_status_past
elastalert_status_error
elastalert_status_silence
elastalert_status
elastalert_status_status
```
Any solution to this?

**Comment by jertel** [comment, marked as answer]
There's no built-in solution. This thread discusses some paths forward: #1299
